<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interfaceContrat</class>
 <widget class="QDialog" name="interfaceContrat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Location">
    <attribute name="title">
     <string>Location</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>691</width>
       <height>501</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Ajouter contrat</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>511</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Durée de la location </string>
       </property>
       <widget class="QDateEdit" name="dateEdit_debutLocation">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit_finLocation">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>50</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>DATE DEBUT:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>30</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>DATE FIN:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>HEURE :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>30</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>HEURE :</string>
        </property>
       </widget>
       <widget class="QTimeEdit" name="timeEdit_heureDebut">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>50</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QTimeEdit" name="timeEdit_heureFin">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>50</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Idagence">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>180</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Idagent">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>180</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>180</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID AGENT :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>180</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID AGENCE :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>260</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>REFERENCE VEHICULE :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Reference">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>260</y>
         <width>281</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>330</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID CLIENT :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Idclient">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>320</y>
         <width>281</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_AjouterContrat">
       <property name="geometry">
        <rect>
         <x>492</x>
         <y>410</y>
         <width>161</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter Contrat</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Modifier contrat</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_Idagent_2">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>200</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>350</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID CLIENT :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ModifieContrat">
       <property name="geometry">
        <rect>
         <x>492</x>
         <y>430</y>
         <width>161</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier Contrat</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>511</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Durée de la location </string>
       </property>
       <widget class="QDateEdit" name="dateEdit_debutLocation_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit_finLocation_2">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>50</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>DATE DEBUT:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>30</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>DATE FIN:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>HEURE :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>30</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>HEURE :</string>
        </property>
       </widget>
       <widget class="QTimeEdit" name="timeEdit_heureDebut_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>50</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QTimeEdit" name="timeEdit_heureFin_2">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>50</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>280</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>REFERENCE VEHICULE :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Reference_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>280</y>
         <width>281</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID AGENCE :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Idagence_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>200</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Idclient_2">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>340</y>
         <width>281</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>200</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID AGENT :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>ID CONTRAT LOCATION A MODIFIE :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_IdcontratM">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>supprimer contrat</string>
      </attribute>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID DU CONTART A SUPPRIMER  :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_IdcontratSupprimer">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>90</y>
         <width>261</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_supprimerContrat">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>380</y>
         <width>171</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer Contrat</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>historique</string>
      </attribute>
      <widget class="QTableView" name="tableContratLocation">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>591</width>
         <height>391</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Assurence">
    <attribute name="title">
     <string>Assurance</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
